update_npm_dependencies.rb

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env ruby
+
+require 'optparse'
+require 'json'
+
+def package_name(npm_module_name)
+  "nodejs-#{npm_module_name}"
+end
+
+def package_dir(npm_module_name)
+  "packages/foreman/#{package_name(npm_module_name)}"
```

Comment:
In the future we might need to make this somewhat more generic but currently `add_npm_package.sh` can't deal with this anyway. Ideally we can use the same for plugins and katello.

Reply:
Yeah, one more option could solve this but as you say, I saw it hardcoded in the other script so I left it as it was for the first iteration.

Reply:
In https://github.com/theforeman/foreman-packaging/pull/2632 @ehelms took the chance to make it configurable :)